Processes/Docs-Contribution-Process-Internal.md

```diff
@@ -0,0 +1,105 @@
+
+
+# Docs Contribution Process - Internal
+
+## Overview
+This document provides instructions for contributors internal to the ENF Foundation on how to contribute new or updated content to EOS Documentation (https://docs.eosnetwork.com).
+
+ You can rely on the DevRel team to perform a thorough edit and review of your content before it is published to the EOS Documentation portal. The publication process includes GitHub workflows to publish Markdown (.md) formatted content.
+
+As a content contributor you are responsible for the accuracy and completeness of your submitted documentation. The DevRel team may contact you for clarification of your contribution, if necessary. You are also responsible to identify future changes for your content and let the DevRel team know of any information maintenance issues.
+
+As a content contributor you are also responsible to review the final submission before it is published. Your review is reflective of the accuracy and completeness of the content.
+
+### When to Use this Procedure
+Use this procedure when you need to create or update content from one or repositories to  [EOS Documentation](https://docs.eosnetwork.com).
+
+### Expected Results
+Fresh, updated content is available for consumption in EOS Documentation.
+
+### Stages of Publication with Contributor Roles Identified
+1. Author: Create branch
+2. Author: Create and name document
+3. Author: Apply template and write content
+4. Author: Submit to DevRel
+5. DevRel: Refine submission according to technical writing best practices
+6. Author: Add changes, if necessary
+7. Author: Final review and approval
+
+
+## Write Content for the EOS Documentation Portal
+Provide content to the DevRel team in Markdown (.md) format.
+
+The ENF Developer Relations team (DevRel) developed a set of templates to assist development of documentation for the EOS ecosystem. The templates are written in Markdown (.md) format and provide a structure for your content contribution. Take advantage of these writing tools to assist with your content contribution. These templates are guidelines for various types of content and will help you organize your content with a common look and feel. These templates are not a substitute for content accuracy and completeness.
+Complete the following steps to apply a template to your content:
+1. Open a new document in your GitHub branch web editor
+2. Open the DevRel repository **Template** Folder
+3. Open the template file for your document type. 
```

Comment:
We don't really need to explain this, as we already have it explained here: https://github.com/eosnetworkfoundation/devrel/tree/main/Templates

Please just link to other explanations instead of having two sources we need to change if we change a process